flink-end-to-end-tests/test-scripts/test_glue_schema_registry_avro.sh

```diff
@@ -25,7 +25,7 @@
 ################################################################################
```

Comment:
I think this entire script can be deleted? Since we have now migrated the test to the java framework?

Reply:
Deleted in the latest commit.